node/cmd/guardiand/adminserver.go

```diff
@@ -879,6 +879,7 @@ func (s *nodePrivilegedService) DumpRPCs(ctx context.Context, req *nodev1.DumpRP
 	rpcMap["polygonRPC"] = *polygonRPC
 	rpcMap["pythnetRPC"] = *pythnetRPC
 	rpcMap["pythnetWS"] = *pythnetWS
+	rpcMap["sei"] = "IBC"
```

Comment:
Will this be the case for all IBC chains?

Reply:
Should this be the wormchain WS?

Reply:
For other chains that are configured for full nodes, I guess it would have that RPC. Sei is only supported over IBC, so that's why it says "IBC". The IBC WS is listed above. I thought it would be nice for it to stand out as something special (IBC) rather than just another RPC.

But we can make it whatever people want.

Reply:
Ah, I see now that I looked harder 😆